pkg/container-disk/container-disk_test.go

```diff
@@ -92,78 +91,88 @@ var _ = Describe("ContainerDisk", func() {
 				Entry("raw disk", "raw"),
 			)
 			It("by verifying error when no disk is present", func() {
+				vmi := libvmi.New()
 
-				vmi := api.NewMinimalVMI("fake-vmi")
-				appendContainerDisk(vmi, "r0")
+				By("Trying to get the volume mount directory on guest and expecting an error")
+				volumeMountDir := GetVolumeMountDirOnGuest(vmi)
+				_, err := os.Stat(filepath.Join(volumeMountDir, "disk_0.img"))
+				Expect(err).To(HaveOccurred())
+				Expect(err).To(MatchError(os.ErrNotExist))
```

Comment:
Does this work? IIUC `MatchError` want's to see a string as second arg in uses like this?

Reply:
This method doesn't work anymore
the `ToMatchError` is undefined in `Expect(err).`
```go
Expect(err).ToMatchError(os.IsExist, "IsExist")]
```
we can achieve the same logic with
```go 
Expect(err).To(MatchError(os.ErrNotExist))
```

Reply:
https://onsi.github.io/gomega/#matcherrorexpected-interface

> If using a function in this way you are required to pass a FUNCTION_ERROR_DESCRIPTION argument to MatchError that describes the function. This description is used in the failure message. For example: Expect(err).To(MatchError(os.IsNotExist, "IsNotExist))